2023/YoctoDevDay/fido-device-onboarding.md

```diff
@@ -0,0 +1,24 @@
+Title:
+FIDO Device Onboarding: Late-binding Provisioning & Tales from the Trenches of
+Bleeding Edge Tech
+
+Abstract:
+This talk explores the role of FIDO Device Onboarding [1] in strengthening the device provisioning
```

Comment:
80 ch wrap?

@TomaszAIR @artur-rs I guess pre-commit could be added for this repo